pages.js

```diff
@@ -1,20 +1,19 @@
 $(function() {
-  $(':checkbox').attr('checked', $.cookie('pjax'))
+  $(':checkbox').prop('checked', sessionStorage['pjax'])
 
-  if ( !$(':checkbox').attr('checked') )
-    $.fn.pjax = $.noop
+  if ( !$(':checkbox').prop('checked') )
+    $.fn.pjax = $.pjax.submit = $.noop
 
   $(':checkbox').change(function() {
     if ( $.pjax == $.noop ) {
-      $(this).removeAttr('checked')
+      $(this).removeProp('checked')
       return alert( "Sorry, your browser doesn't support pjax :(" )
     }
-
-    if ( $(this).attr('checked') )
-      $.cookie('pjax', true)
+    if ( $(this).prop('checked') )
+      sessionStorage['pjax'] = true
     else
-      $.cookie('pjax', null)
+      sessionStorage['pjax'] = ''
 
     window.location = location.href
```

Comment:
```js
    if ( $(this).prop('checked') ) {
      $.pjax.enable()
      sessionStorage['pjax'] = true
    } else {
      $.pjax.disable()
      sessionStorage['pjax'] = ''
    }
    //window.location = location.href               
```
I tried this, but I got an error with following steps:
1. Reload browser after turning on pjax checkbox.
2. Turn off pjax.
3. Click any link.

```
[Error] TypeError: undefined is not an object (evaluating '$.pjax.defaults.version')
	success (jquery.pjax.js, line 254)
	fire (jquery-1.11.2.js, line 3143)
	fireWith (jquery-1.11.2.js, line 3255)
	done (jquery-1.11.2.js, line 9309)
	callback (jquery-1.11.2.js, line 9713)
```